x-pack/legacy/plugins/siem/public/containers/case/api.ts

```diff
@@ -130,3 +130,25 @@ export const patchComment = async (
   await throwIfNotOk(response.response);
   return convertToCamelCase<CommentResponse, Comment>(decodeCommentResponse(response.body));
 };
+
+export const deleteCases = async (caseIds: string[]): Promise<boolean> => {
+  const response = await KibanaServices.get().http.fetch<string>(`${CASES_URL}`, {
+    method: 'DELETE',
+    asResponse: true,
+    query: { ids: JSON.stringify(caseIds) },
+  });
+  await throwIfNotOk(response.response);
+  return response.body === 'true' ? true : false;
```

Comment:
I think is better to return 204 and an empty body than booleans.

Reply:
i agree, but lets do that in a new pr. i really also dont like true as a string ever